resources/windows/wix/customactions.wxs.erb

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="windows-1252"?>
```

Comment:
Its a good point - we probably don't need to make this as a template as I think we can use properties (the properties file would benefit from a template)